packages/console/src/components/Sidebar/components/Item/index.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '@/scss/underscore' as _;
+
+.row {
+  display: flex;
+  align-items: center;
+  margin: 0 _.unit(5) 0 _.unit(2);
+  padding: _.unit(3) _.unit(4);
+  color: var(--color-on-surface-variant);
+  border-radius: _.unit(2);
```

Comment:
I think this whole `underscore` thing is a little bit cumbersome. What's good of using this approach instead of simply using `px` or `rem` units?

Reply:
- `px` will be a trouble if we want to change the multiplier foundation in the future
- i tired `rem` with `12px` in the body before but i'll be tricky to make sizes like 16 (`1.25rem`)
any ideas?

Reply:
We can set 1 rem to 10px by using:
```
html { font-size: 62.5%; }
```
This trick will help calculate the rest of the elements, e.g. 16px would be 1.6rem

Reply:
looks weird, comparing `_.unit(3)` will compile to `12px` which is more accurate